Detectors/TPC/reconstruction/test/testGPUCATracking.cxx

```diff
@@ -21,12 +21,14 @@
 #include "DataFormatsTPC/TrackTPC.h"
 #include "DataFormatsTPC/ClusterNative.h"
 #include "DataFormatsTPC/ClusterNativeHelper.h"
+#include "TPCBase/Mapper.h"
```

Comment:
Just a note: using the mapper is fine in principle, but a lot of the mapper functionality is available in `SliceTracker/GPUTPCGeometry.h`, which is part of the `GPUParam`. I would take as much as possible from there.